src/Reactor.jl

```diff
@@ -310,7 +327,7 @@ function Reactor(domain::T, y0unlumped::Array{W1,1}, tspan::Tuple, reducedmodelm
     if modelingtoolkit
         sys = modelingtoolkitize(ode)
         jac = eval(ModelingToolkit.generate_jacobian(sys)[2])
-        odefcn = ODEFunction(dydt; jac=jac, paramjac=jacp!)
+        odefcn = ODEFunction(dydt; jac=jac, paramjac=jacp!, jac_prototype=jac_prototype)
```

Comment:
Is `modelingtoolkitize` tested in our test? Maybe we should add a test

Reply:
We should, but we need a very small mechanism to make it fast.